assets/component-menu-drawer.css

```diff
@@ -37,10 +41,19 @@ menu-drawer > details[open] > summary::before {
   top: 100%;
   width: calc(100vw - 4rem);
   padding: 0;
-  border: 0.1rem solid rgba(var(--color-foreground), 0.2);
-  border-left: 0;
+  border-width: 0 var(--popup-drawer-border-width) 0 0;
```

Comment:
With a border it looks a bit weird: 
![](https://screenshot.click/21-11-fcq2d-wpdrb.png)

Reply:
I think I preferred with no border on the left side only